util/request_source.go

```diff
@@ -52,12 +56,29 @@ func (r *RequestSource) SetRequestSourceType(tp string) {
 	r.RequestSourceType = tp
 }
 
+// SetRunInBackground sets the request to run in background.
+func (r *RequestSource) SetRunInBackground(background bool) {
+	r.RunInBackground = background
+}
+
+// RequestSourceOption is used to set the request source.
+type RequestSourceOption func(*RequestSource)
+
+// RequestRunInBackground sets the request to run in background.
+var RequestRunInBackgrounOption = func(rs *RequestSource) {
```

Comment:
```suggestion
var RequestRunInBackgroundOption = func(rs *RequestSource) {
```
When reviewing the code, I noticed WithResouceGroupName missed a r, can u fix it BTW? :)